permissions/plugin-date-parameter.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "date-parameter"
+paths:
+- "me/leejay/jenkins/date-parameter"
+developers:
+- "leejaycoke"
+---
```

Comment:
I've found a problem. And it was fixed.
It's '---' string at last line.
But I do not know if "This" means this.